app/views/partials/_comment.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class='comment'>
+  <h2><%= comment.user.name %>: </h2>
+  <p><%= comment.text %></p>
+  <p><%= comment.created_at.strftime('%a %d %b %Y') %></p>
```

Comment:
- [ ] Unable to create comments via form views due to missing comment form view, and route for new comments. Kindly ensure that comments can be attached to a post via form views successfully.

![image](https://github.com/Movgto/my-blog-app/assets/12745474/cbaed290-0b18-4d66-84d5-fe8bb9f4d286)

Reply:
You can make comments and give likes on the post view, do I need to make a different page for the comments and likes?

Reply:
It looks like this @rloterh 
![image](https://github.com/Movgto/my-blog-app/assets/107658605/5a6d8b0f-54cb-48b4-bd2e-64e7596e229d)

Reply:
Hi @Movgto 

Thanks for sharing your working version, however, it's rather appearing as below per you cloned project after liking:

![image](https://github.com/Movgto/my-blog-app/assets/12745474/dc70198b-796f-4b84-8da5-2e656be11ff9)

I think you should check and ensure that all latest and working changes are pushed to the repository online.

Cheers!

Reply:
@rloterh The comment field was inside the comments container, and I made it to not show if there were no comments yet, so that's why the comment field is not visible when there are no comments in the post, but I already fixed that on my last commit on this branch

Reply:
Hi @Movgto 

Thanks for letting me know 👍 However, in this activity, we want users to able to comment on a users post using form views, not on the console side. Let's therefore show the comment form view (not particularly the comment itself) so that users can easily attach a comment to a post via form views.

Cheers!

Reply:
@rloterh But that's already available on the application, the users can comment on posts with a form as shown on the screenshot I sent previously, it's not necessary to use the console for that, and I fixed a bug that was making the comment form not visible when a post didn't have any comments yet, so it should be there if you pull the last update :(